### DISCUSSION TITLE
context.contextActivities.category

### DISCUSSION THREAD

**Discussion by sfraysse** [gaia-x-dases/xapi-virtual-classroom#2] [General]
The `context.contextActivities.category` prop should contain an activity identifying the virtual classroom xAPI profile.

In the current examples, these activities identify a template, not a profile:
```
http://schema.dases.eu/xapi/profile/virtual-classroom/templates/initialized
http://schema.dases.eu/xapi/profile/virtual-classroom/templates/terminated
...
```

We should use something like:
```
http://schema.dases.eu/xapi/profile/virtual-classroom
```
Or even better:
```
https://w3id.org/xapi/virtual-classroom
```

**Comment by quitterie-lcs** [comment]
Indeed you're right, that's explicitly specified[ in the spec](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#details-15). I read it too fast
